.gitignore

```diff
@@ -9,5 +9,9 @@ web/profiles/contrib
 # Ignore Drupal's file directory
 web/sites/default/files
 
+# Ignore environment specific settings
+web/sites/default/services.yml
```

Comment:
My reasoning was that it can be used to turn on things like Twig debugging which you don't want to have accidentally deployed to production.